modules/persistent-storage-csi-cloning-provisioning.adoc

```diff
@@ -5,22 +5,20 @@
 [id="persistent-storage-csi-cloning-provisioning_{context}"]
 = Provisioning a CSI volume clone
 
-Provisioning of a CSI volume clone is triggered by the creation of a cloned persistent volume claim (PVC) API object. The cloned PVC is pre-populated with the contents of another PVC. After creation, the cloned volume adheres to the same rules as any other persistent volume.
+When you create a cloned persistent volume claim (PVC) API object, you trigger the provisioning of a CSI volume clone. The clone pre-populates with the contents of another PVC, adhering to the same rules as any other persistent volume.
+
+The one exception of cloned provisioning is that you must add a `dataSource` that references an existing PVC in the same namespace.
```

Comment:
I had to re-read this line a few times to know what you were talking about that this was an exception to. I'd suggest adding this to the paragraph above, since you're talking about that it's the same rules as any other PV, with this exception.